routes/messages/commands/amazon_link.js

```diff
@@ -4,7 +4,15 @@ const Users = require('../../../models/db/users.js');
 const getUserInfo = require('../util/asycFunctions');
 
 module.exports = async event => {
-  const id = 1; //need a dynamic way to pull ids
+  // we have facebook id, 
+  // from facebook id , we can get the userid,
+  // can dynamically pass user id 
+  // const user = await Users.retrieveOrCreate({ facebook_id: event.sender.id });
+  //console.log(user.id, 'user');
+  //console.log(event.sender.id, 'sender id');
+  //const id = user.id;
+  //will dynamically pass id 
+  const id = 1;
```

Comment:
This is the book ID, not the user ID.  We currently don't have a way to dynamically get the book ID since we can only pass along a string.  It's not impossible, we just haven't done it yet since we're only working with one book but it's a problem to be solved as we expand.

Reply:
this is what I was thinking and was jotting a note for myself.